example/30-worker.yaml

```
@@ -84,5 +84,8 @@ spec:
     volume:
       type: pd-standard
       size: 20Gi
+    # workerconfig is used to specify the interface of gcp local ssd
+    dataVolumes:
+      - type: SCRATCH
```

Comment it to show it's just a possible option but disabled by default

```suggestion
  # dataVolumes:
  # - type: SCRATCH
```

---

Done